src/include/block.h

```diff
@@ -302,6 +302,7 @@ struct __wt_block {
     uint64_t compact_pages_rewritten;          /* Pages rewritten */
     uint64_t compact_pages_rewritten_expected; /* The expected number of pages to rewrite */
     uint64_t compact_pages_skipped;            /* Pages skipped */
+    uint64_t compact_prev_pages_rewritten;     /* Pages rewritten during the previous iteration */
```

Comment:
Is `compact_prev_pages_rewritten` initialized? You might want to zero it in `__wt_block_compact_start`.

Reply:
good point! added https://github.com/wiredtiger/wiredtiger/pull/10447/commits/41055f53ac889416a650116dc00a6c12b749744b